api/src/main/java/org/pmiops/workbench/api/ProfileController.java

```diff
@@ -260,7 +260,7 @@ private DbUser initializeUserIfNeeded() {
           userService.createUser(
               profile.getGivenName(),
               profile.getFamilyName(),
-              googleUser.getPrimaryEmail(),
+              gSuiteUsername,
```

Comment:
this is the same value we use to create the googleUser.primaryEmail just above here